src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -837,6 +858,24 @@ bool primitive_inst::update_impl() {
     }
 
     if (!_node->is_type<data>() && !(_node->is_type<mutable_data>() && _node->get_dependencies().empty())) {
+#ifdef ENABLE_ONEDNN_FOR_GPU
+        if (get_node().get_preferred_impl_type() == impl_types::onednn) {
+            auto attrs_onednn = std::make_shared<dnnl::primitive_attr>();
+            std::vector<cldnn::fused_primitive_desc_onednn> fused_desc_onednn;
+            cldnn::create_onednn_primitive_attributes(get_node(),
+                                                        _impl_params->fused_desc,
+                                                        attrs_onednn,
+                                                        fused_desc_onednn,
+                                                        _impl_params.get());
```

Comment:
did you have a chance to check runtime host overhead of this? If it is heavy, we may review to take different approach..

Reply:
I think we can check more detailed perf & optimize,  once we finalize the initial implementation (this PR). 
We can do step by step because currently we dont observe any regression at least.

Reply:
As Taylor's comments, I will check more detailed perf & optimization including this part after initial implementation.